src/Moderator.cpp

```diff
@@ -69,19 +69,23 @@ unsigned Moderator::get_search_depth(const DomineeringState& state) const {
     unsigned game_moves = state.getNumMoves();
     unsigned depth;
 
+    if (searcher.get_time_left() > 20.0) {
```

Comment:
The value looks legitimate. Just create a `static constexpr float TIME_LIMIT = 20;` or something. Arbitrary number (other than 42) appearing in code almost never looks good.